### ISSUE TITLE
Change language on View-only Dashboard Access screen

### ISSUE THREAD

**Issue by eclarke1** [google/site-kit-wp#5441]
## Feature Description

Bug bash issue: https://app.asana.com/0/1202258919887896/1202444327251138 please see Asana for background

Current language under the header is "Get insights about how people find and use your site, how to improve and monetize your content, directly in your WordPress dashboard" which is not a complete sentence.

Change to:
"Get insights about how people find and use your site as well as how to improve and monetize your content, directly in your WordPress dashboard"

![image.png](https://images.zenhubusercontent.com/5e2ad3803c374569231d8cd7/fb38294a-1c50-4d9e-9ddf-648388a436b5)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by wpdarren** [comment]
@mohitwp @makiost @eclarke1 I might be fussy here but I am not loving the paragraph layout. I feel we need to have a space between the first and second paragraph to make it easier to read. What are your thoughts here?

**Comment by wpdarren** [comment]
@mohitwp @makiost this might be me, being overly fussy, but noticed that on lower resolutions and also smaller screen sizes, the two paragraphs are hard to read and feel that a space needs to be introduced between the two paragraphs. I noticed this while testing the feature flag and it jumped out at me.  Thoughts welcome here. 

Desktop 

![image](https://user-images.githubusercontent.com/73545194/177928116-89f28a21-1567-40af-947a-669f343cbdcb.png)


Mobile 
<details>
 <summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/73545194/177928330-4c61f8e0-e32c-48a4-9b87-b69cc4d5fb91.png)

</details>

**Comment by eclarke1** [comment]
I agree to having a space in between those 2 paragraphs, wonder if @marrrmarrr can also comment to this.
I notice this is in Approval though, so might be worth keeping it in the QA column until that decision is made. Moving it back there now.
@marrrmarrr let us know if you agree to adding a space between those 2 paragraphs please.

**Comment by eclarke1** [comment]
@marrrmarrr just a friendly ping on this one if I may please, it's due to go in this release so would be great to get your thoughts here.

**Comment by aaemnnosttv** [comment]
Let's fix it, it should be a very quick fix. We just need to remove the `googlesitekit-setup__description` on the second paragraph.
![image](https://user-images.githubusercontent.com/1621608/178343186-5eaa69a2-082d-406b-b03a-ffcae5b2adec.png)

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- Text was changed as per the AC
- The two paragraphs have a space which make it easier to read on lower resolution/smaller screen sizes. 

![Uploading image.png…]()
